src/Windows/local_cache.cpp

```diff
@@ -194,6 +194,23 @@ static std::wstring get_file_name(const std::string& id)
     return g_cache_dirname + L"\\" + sha256(id);
 }
 
+std::string get_cached_file_location(const std::string& id){
+	std::wstring fileName = get_file_name(id);
+	//Turning a wstring into a string in this way truncates the value of the char, 
+	//which can have unexpected results when it involves non-ASCII characters
+	//We don't use non-ASCII characters for the default cache path, so that shouldn't be an issue
+	return std::string(fileName.begin(), fileName.end());
+}
+
+std::string get_last_cache_read_expiry(){
+	/*std::string result("Last cache read expiry is unset. This could be because there's been no cache reads yet, it has been read without checking for cache expiry or an error during cache read before cache expiry is checked.");
+	if(last_cache_read_expiry != -1){
+		result = "Last cache read expiry value is " + std::to_string(last_cache_read_expiry) + " and it was read when time(nullptr) value was " + std::to_string(last_cache_read_time);
+	}
+	return result;*/
```

Comment:
I still think we don't need this function but if we finally keep it, please remove commented code.